package.json

```diff
@@ -85,6 +85,13 @@
     ],
     "main": "./main.js",
     "contributes": {
+        "x-azResources": {
+            "activation": {
+                "onFetch": [
```

Comment:
Have we considered `onResolve`?

Reply:
I'm not totally savvy to how this works, but I figured that the fetch would always happen before the resolve.  But I'm totally okay to add it to the manifest.

Reply:
I'd suggest using `onResolve` anytime possible--since it's the latest and most accurate activation--and only using `onFetch` (or even earlier, `onView`) if necessary.

Reply:
Having both `onFetch` and `onResolve` is equivalent to `onFetch` only, since it will happen first.